index.md

```diff
@@ -184,7 +184,7 @@ Contributing
 : [Contribution overview](https://github.com/cockpit-project/cockpit/wiki/Contributing)
 : [Get the source](https://github.com/cockpit-project/cockpit)
 : [Join the mailing list](https://lists.fedorahosted.org/archives/list/cockpit-devel@lists.fedorahosted.org/)
-: [IRC #cockpit on Freenode.net](irc://irc.freenode.net:6667/cockpit)
+: [IRC #cockpit on libera.chat](https://libera.chat/) -- [kiwiirc web client](https://kiwiirc.com/nextclient/irc.libera.chat/#cockpit)
```

Comment:
This is weird in the footer. (What we have previously was also weird in the footer.)

Perhaps we need an IRC page with instructions and click-to-opt-in iframe with the web chat? It could just be a page with IRC instructions to start though.

Reply:
Or it could be a block in contribute and we could link to that. (We have a contact in the privacy; it could be a copy/paste/adjust from that.)

Reply:
Good point, updated to keep the original form for now.